src/txHashBuilder_test.c

```diff
@@ -216,7 +216,7 @@ static struct {
 };
 
 static struct {
-	const char* rewardAccount;
+	const char* rewardAddress;
```

Comment:
worth noting that the original test vector here was 29 bytes, so it was a proper withdrawal with the reward address indeed